oak-doc/src/site/markdown/security/accesscontrol/default.md

```diff
@@ -129,8 +129,8 @@ found in section [Restriction Management](../authorization/restriction.html).
 <a name="representation"></a>
 ### Representation in the Repository
 
-All access control policies defined with an Oak repository are stores child of
-the node they are bound to. The node type definition used to represent access control content:
+All access control policies defined with an Oak repository are stored as child of
```

Comment:
hi @kwin , thanks for spotting the mistake!
given that the statement is only true for the default authorization model, it might be more accurate and a bit less prone to misunderstandings if we would make this clear in this sentence.... e.g. something like

`Access control policies created and modified using the default authorization model are stored as child of`

because the statement that this applies for _all policies defined with an oak repository_ is quite obviously not accurate. btw. it is not even accurate for the default model which also defines 'named policies' for the configured readable paths..... the statement only applies for those policies that are applicabe/modifiable through JCR ac management API.

wdyt

Reply:
Done